reference/types/floating_point_number.md

```diff
@@ -10,15 +10,42 @@ Programming languages may name their floating point type to the amount of _preci
 - Quadruple (occupying 128 [bits][type-bit])
 - Octuple (occupying 256 [bits][type-bit])
 
-# Implementations
+## What to cover
 
-- [C# (basic)][implementation-csharp-basic]
-- [C# (advanced)][implementation-csharp-advanced]
+A student may have no understanding of floating point numbers as more than numbers with decimal points.
+
+- Explain that a floating point number is not *just* a number with a decimal place.
+- Explain what floating point types are used in your langauge, and when to use them.
+- Ensure that type conversations are understood if appropriate (e.g. rounding, precision-changing)
+- Ensure that students know how to compare numbers to each other.
+
+## Exercises
+
+### Production Line Analysis
+
+This exercise asks students to write code to analyse the production of an assembly line. It covers:
+- arithmetic
+- numeric comparison / conditionals
+- parsing between integers and floating point numbers.
```

Comment:
This does a similar job as I'm not trying to do with my `concepts exercises` list: https://github.com/exercism/v3/issues/212

In this list I am writing down all the concepts needed for an exercise, not which one it introduces and which ones are prerequisites. Because which ones it introduces is very dependent on which other `concept exercises` are before which can quickly change. E.g. when we figure out it might be worthwhile to introduce another exercise for a basic concept or the exercise before it changes and also needs `conditionals` now. Suddenly conditionals are not taught by this exercise any more but by the one before.

Since this is also something that might vary from language to language, I'd consider this to be a list of all concepts needed to solve this exercise, not a list of concepts it teaches. Or we only list here the concepts it focuses on, and then `conditionals` should not be part of that list -- at least not in a general file for all languages.

Reply:
Another good option... 🤔

Reply:
@iHiD What do you think about the [above suggestion](https://github.com/exercism/v3/pull/342#discussion_r370986489) I made?

Reply:
I don't know tbh.

The aim of this is that a maintainer could think "I need an exercise that covers X and Y" and then go and look in these files and spot one that seems to match what they need. But also that people can share stories (which is a big part of the writing effort). So maybe we have:
```
### Production Line Analysis
Story and general info

#### C# Implementation
- concept 1
- concept 2

I'm unclear on how best to order it to achieve that.


Again, I'm not sure. Maybe

Reply:
So the thing I'm aiming for is for people to be able to see how a particular language has interpreted the Concepts for this exercise. What I am slightly afraid of, is that we'll have these "this exercise shouldn't teach Concept X because this wouldn't work for my language." So essentially a bit like  the issues we've had with the prob-specs repo.

Reply:
Yeah. Makes sense. Although I figured you are the "owner" of this exercise. But yes, maybe listing your concepts in your language makes sense.

Reply:
@iHiD Want me to update this PR to the format I suggested?

Reply:
Yeah - let's see what that looks like :)